argo-event/values.yaml

```diff
@@ -0,0 +1,114 @@
+
+# Name, required
+# example
+# name: use-case-1
+
+# Labels for eventbus, event source and sensor, optional
+labels: {}
+
+# For event bus
+eventbus:
```

Comment:
Add support for maxAge https://github.com/argoproj/argo-events/blob/stable/api/event-bus.md#nativestrategy

Reply:
After short discussion, we will use the set the default value of maxAge as 168h first (which is the default value from argo-event), but it may need to change the maxAge to a smaller value

ref: https://github.com/argoproj/argo-events/blob/bf455eac8bb82b2d443f856f50e91943d78ba77c/manifests/install.yaml#L329

Reply:
Forgot to mention, please also support duplicates and max_file_store.